### DISCUSSION TITLE
震惊，店长竟然是这样的人！

### DISCUSSION THREAD

**Discussion by biliqide** [share121/inter-knot#1721] [其他]
今天在网上冲浪，发现有
![1723456367347](https://github.com/user-attachments/assets/7db114ba-cd15-4e03-ab70-670d45a0cfc6)
变态跟踪狂的新闻，仔细一看，竟然是店长

**Comment by lllliufeng** [comment]
啊？真的吗
有录像带吗

**Comment by aizai1173** [comment]
星徽骑士怎么跟艾莲一起走？

**Comment by WORLD-executeME** [comment]
假的吧

**Comment by yutong903** [comment]
不可能，昨天店长和我卓爱了一整天，不可能有时间出去（

**Comment by gaojiaobei** [comment]
可怕，真是人不可貌相啊
